app/src/main/res/xml/preferences_master.xml

```diff
@@ -403,6 +403,12 @@
                     android:icon="@drawable/ic_code_black_24dp"
                     android:key="@string/pref_key__markdown__disable_code_block_highlight"
                     android:title="@string/disable_code_block_highlight" />
+                <CheckBoxPreference
```

Comment:
I'm fine with it being easy to enable/disable in code, but we don't need a settings option for it. It's fine to have this appsettings method and just return true. And have no settings.xml option.

Reply:
I think we should either
1. Have a settings option for it
2. Do not enable auto-update at all (manual update is still possible with long-press)

If we permanently enable this, then it will be impossible to edit lists without changing the numbering.

For example, I sometimes like to leave my lists as:
```
1. foo
1. bar
1. baz
```
Cant do this if the auto-renumber is always true as entering a new item will renumber the entire list.

Reply:
Another issue is there may be performance issues on long lists and older phones.